README.md

```diff
@@ -1,602 +1,387 @@
+[Capybara]: https://github.com/teamcapybara/capybara
+[discourages controller specs entirely]: rspec.info/blog/2016/07/rspec-3-5-has-been-released/#rails-support-for-rails-5
 
-```ruby
-expect(:get => "/widgets/1/edit").not_to be_routable
-```
-
-### `have_http_status`
-
-- Passes if `response` has a matching HTTP status code
-- The following symbolic status codes are allowed:
-  - `Rack::Utils::SYMBOL_TO_STATUS_CODE`
-  - One of the defined `ActionDispatch::TestResponse` aliases:
-    - `:error`
-    - `:missing`
-    - `:redirect`
-    - `:success`
-- Available in controller, feature, and request specs.
-
-In controller and request specs, apply to the `response` object:
-
-```ruby
-expect(response).to have_http_status(201)
-expect(response).not_to have_http_status(:created)
-```
-
-In feature specs, apply to the `page` object:
-
-```ruby
-expect(page).to have_http_status(:success)
-```
-
-## `rake` tasks
+## Contributing
 
-Several rake tasks are provided as a convenience for working with RSpec. To run
-the entire spec suite use `rake spec`. To run a subset of specs use the
-associated type task, for example `rake spec:models`.
+- [Build details](BUILD_DETAIL.md)
+- [Code of Conduct](CODE_OF_CONDUCT.md)
+- [Detailed contributing guide](CONTRIBUTING.md)
 
-A full list of the available rake tasks can be seen by running `rake -T | grep
-spec`.
+Once you’ve cloned the repo and [set up the environment](DEVELOPMENT.md),
+you can run the specs and Cucumber features, or submit a pull request.
 
-### Customizing `rake` tasks
+## See Also
 
-If you want to customize the behavior of `rake spec`, you may [define your own
-task in the `Rakefile` for your
-project](https://www.relishapp.com/rspec/rspec-core/docs/command-line/rake-task).
-However, you must first clear the task that rspec-rails defined:
+### RSpec base libraries
 
-```ruby
-task("spec").clear
-```
+* <https://github.com/rspec/rspec>
+* <https://github.com/rspec/rspec-core>
+* <https://github.com/rspec/rspec-expectations>
+* <https://github.com/rspec/rspec-mocks>
 
+### Recommended third-party extensions
 
-## Also see
+* [FactoryBot](https://github.com/thoughtbot/factory_bot)
+* [Capybara](https://github.com/jnicklas/capybara)  
+  (Included by default in Rails 5.1+.
+  Note that [additional configuration is required][] to use the Capybara DSL
+  anywhere other than system specs and feature specs.)
 
-* [https://github.com/rspec/rspec](https://github.com/rspec/rspec)
-* [https://github.com/rspec/rspec-core](https://github.com/rspec/rspec-core)
-* [https://github.com/rspec/rspec-expectations](https://github.com/rspec/rspec-expectations)
-* [https://github.com/rspec/rspec-mocks](https://github.com/rspec/rspec-mocks)
+  [additional configuration is required]: http://rubydoc.info/gems/rspec-rails/file/Capybara.md
 
-## Feature Requests & Bugs
+### RSpec style guides
 
-See <http://github.com/rspec/rspec-rails/issues>
+* [Better Specs](http://www.betterspecs.org/)
```

Comment:
Better specs are not recommended by us and contain many antipatterns, I will not merge a PR with this in.